cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -359,6 +349,19 @@ func gitIndex(c gitIndexConfig, o *indexArgs) error {
 	return nil
 }
 
+func priorBranches(c gitIndexConfig, o *indexArgs) ([]zoekt.RepositoryBranch, error) {
```

Comment:
I'd prefer if this was inlined above since only it's only used in one place. It's not a big deal though.

Reply:
the reason I pulled this out is to try and make the size of the index function smaller which I hope improves readability of it. WDYT?